### PR TITLE
[MRG] Change minimum numpy ver to 1.11.0  for NiLearn compatibility

### PR DESCRIPTION
**Pull request by kchawla-pi**

### CODE REVIEW

File: continuous_integration/circle_ci_test_doc.sh [hunk 1]
```diff
@@ -1,10 +0,0 @@
-#!bin/bash
```
Comment (bthirion) [inline continuous_integration/circle_ci_test_doc.sh]: Why do you need to remove that ?

**Comment by bthirion** [comment]
LGTM then.

**Comment by kchawla-pi** [comment]
Are you still getting that error?

**Comment by bthirion** [comment]
I can only check that  at NSP :-(

**Comment by bthirion** [comment]
If nobody protests I'll merge within 1hour.

**Pull request merged**